beacon-chain/rpc/debug/block.go

```diff
@@ -42,7 +43,7 @@ func (ds *Server) GetInclusionSlot(ctx context.Context, req *pbrpc.InclusionSlot
 	ds.GenesisTimeFetcher.CurrentSlot()
 
 	// Attestation has one epoch to get included in the chain. This blocks users from requesting too soon.
-	epochBack := uint64(0)
+	epochBack := types.Slot(0)
```

Comment:
`epochBack` is a slot? What does this name even mean? :-P

Reply:
Yes, it is a slot. The `epochBack` holds slot number of the previous epoch, it is inited to zero, then calculated.